migration-guides/10.0.md

```diff
@@ -0,0 +1,58 @@
+# Major 10.0 Migration guide
+> [!IMPORTANT]
+> Please make sure to first follow the ["Upgrade new version guide"](https://github.com/AmadeusITGroup/otter/blob/release/main/docs/core/UPGRADE_NEW_VERSION.md) before going through these steps.
+
+
+## Personalization
+From version 10 of @o3r, the components' types must be set with the ```@O3rComponent``` decorator instead of the following
```

Comment:
```suggestion
Starting from version 10 of @o3r, the types of the components types must be set with the ```@O3rComponent``` decorator instead of the following
```